client/src/components/app.tsx

```diff
@@ -90,6 +99,21 @@ class App extends React.Component<Props> {
                         viewportRef={viewportRef}
                         key={graphRenderCounter}
                       />
+                      {scatterplotXXaccessor && scatterplotYYaccessor && (
```

Comment:
Makes more sense to put these at the app level instead of hiding them somewhere arbitrary